.gitignore

```diff
@@ -21,6 +21,10 @@
 /.build
 /Packages
 
+# Commonly used name for build products for llbuild-analyze.
+/products/build
+/products/llbuild-analyze/.build
```

Comment:
Why don't we just change the `/.build` a couple lines up to be `.build`

Reply:
I just had a case where a similar situation caused a lot of confusion but I guess it would be fine due to the `.` prefix.